packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -48,8 +48,29 @@ export function UserHeaderItem(props: PropsWithChildren<UserHeaderItemProps>) {
     const [showAboutModal, setShowAboutModal] = useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
-    const AccountIcon = user.impersonated ? <Account color="info" /> : <Account />;
-
+    const AccountIcon = user.impersonated ? (
```

Comment:
Adding the second user-icon, when impersonating seems to be done in this PR as well: https://github.com/vivid-planet/comet/pull/2650
I'm not sure in which task this was supposed to be done... 🤔

Reply:
Thank you - I didn't know that! 
I talked to @fichtnerma and we decided that I add the styling of the second user-icon, as it was intended in the tasks. He will update his branch afterwards.

I also updated my code, using AvatarGroup now as suggested by @johnnyomair: 0de8e1b9835e45fc1e2e3c6d486b0a9f909373bd

@jamesricky: I couldn't find a way to set the borders correctly without using !important. Do you know any way to avoid this?

Reply:
Unfortunately `AvatarGroup` styles the `Avatar` by default using a nested selector. 

You can override this using the `&&` selector to increase your specificity. 
It's not pretty but will cause less problems than `!important`. 

```ts
const StyledAvatar = styled(Avatar)`
    width: 32px;
    height: 32px;
    background-color: ${({ theme }) => theme.palette.grey[900]};

    && {
        border: 1px solid ${({ theme }) => theme.palette.grey[400]};
    }
`;
```

Reply:
0401a6793e1d7e0963be4b13a4c904bc745f9f3f